src/Simulation/Simulators.Tests/CoreTests.cs

```diff
@@ -44,18 +62,18 @@ public void Borrowing()
 
                 BorrowingTest.Run(s).Wait();
 
-                var tracer = Helper.GetTracer<(long, Qubit)>(s);
+                var tracer = OperationsTestHelper.GetTracer<(long, Qubit)>(s);
 
-                var testOne = new System.Action<int, OperationFunctor, (int, Qubit)>((callsCount, variant, info) =>
+                var testOne = new System.Action<int, OperationFunctor, (int, Qubit)>((int callsCount, OperationFunctor variant, (int, Qubit) info) =>
                 {
                     var (available, q) = info;
                     Assert.Equal(callsCount, tracer.Log.GetNumberOfCalls(variant, (available, q)));
                 });
 
-                var testOneBody = new System.Action<int, (int, Qubit)>((callsCount, info) => testOne(callsCount, OperationFunctor.Body, info));
-                var testOneAdjoint = new System.Action<int, (int, Qubit)>((callsCount, info) => testOne(callsCount, OperationFunctor.Adjoint, info));
-                var testOneCtrl = new System.Action<int, (int, Qubit)>((callsCount, info) => testOne(callsCount, OperationFunctor.Controlled, info));
-                var testOneCtrlAdj = new System.Action<int, (int, Qubit)>((callsCount, info) => testOne(callsCount, OperationFunctor.ControlledAdjoint, info));
+                var testOneBody = new System.Action<int, (int, Qubit)>((int callsCount, (int, Qubit) info) => testOne(callsCount, OperationFunctor.Body, info));
+                var testOneAdjoint = new System.Action<int, (int, Qubit)>((int callsCount, (int, Qubit) info) => testOne(callsCount, OperationFunctor.Adjoint, info));
+                var testOneCtrl = new System.Action<int, (int, Qubit)>((int callsCount, (int, Qubit) info) => testOne(callsCount, OperationFunctor.Controlled, info));
+                var testOneCtrlAdj = new System.Action<int, (int, Qubit)>((int callsCount, (int, Qubit) info) => testOne(callsCount, OperationFunctor.ControlledAdjoint, info));
```

Comment:
Why are the explicit lambda parameter types needed now?

Reply:
Hm, I didn't add them. Maybe I picked them up from another file version or one of the tools did? I'll remove them again.